rules/tax-invoice-vs-eftpos-receipt/rule.md

```diff
@@ -20,83 +20,75 @@ It is important to distinguish the difference between a Tax Invoice and Eftpos P
 
 ![](Picture1.png)
 
+The primary purpose of a Tax Invoice is to provide a comprehensive record of a transaction for tax and accounting purposes.
+While an Eftpos Payment Receipt is like a digital high-five after you've paid for something using Eftpos. It's a record saying, "Yep, the payment went through!"
+
+If you're not sure which papers you need, just keep everything. It's way better to have too many records than miss out on something important.
+
 ### Tax Invoice
 
 An official document issued by a seller to a buyer, detailing the products or services provided along with their respective prices.
 It is a legal requirement for businesses to issue tax invoices for taxable sales.
 A tax invoice would normally have the following:
 
-Business Information
+**Business Information**
 
-* Seller's info - name, address, and contact details.
-* Buyer's info - name and address.
+* Seller's info - Name, address, and contact details
+* Buyer's info - Name and address
 
-Invoice Details
+**Invoice Details**
 
-* Invoice number.
-* Date – when the invoice created.
+* Invoice number
+* Date – When the invoice was created
 
-Product/Service Information
+**Product/Service Information**
 
-* Description - the products or services.
-* How much - Quantity, unit price, and total amount.
+* Description - The products or services
+* How much - Quantity, unit price, and total amount
 
-Tax Information
+**Tax Information**
 
-* Tax - Goods and Services Tax (GST) details.
-* Total amount - including GST.
+* Tax - Goods and Services Tax (GST) details
+* Total amount (including GST)
 
-![Figure:Example of Tax Invoices](Tax-Invoices.jpg)
+![Figure:Example of Tax Invoices](tax-invoice-example-au.1678919047955.avif)
```

Comment:
Hi @JimmyChenSSW 

I can't see this `.avif` image...

- [ ] 1. Can you convert to JPG, upload again, and update the path?

Reply:
Done